libshpool/src/daemon/server.rs

```diff
@@ -544,11 +548,12 @@ impl Server {
         // create a slot to store our reply so we can do
         // our IO without the lock held.
         let reply = {
+            let _s = span!(Level::INFO, "lock(shells)").entered();
```

Comment:
ditto, should we make the name more unique?

Reply:
same answer

Reply:
yep sgtm :)